h/services/group.py

```diff
@@ -21,6 +21,11 @@ def __init__(self, session, user_fetcher):
         self.session = session
         self.user_fetcher = user_fetcher
 
+    def fetch(self, pubid):
+        """Fetch a group by ``pubid``"""
```

Comment:
A more PEP257 docstring would be `"""Return the group with the given ``pubid`` or ``None``."""`